### DISCUSSION TITLE
Reconnecting to another metamask account

### DISCUSSION THREAD

**Discussion by apexof** [wevm/wagmi#605] [Ideas]
If I was connected once, then disconnected, then subsequent connections occur automatically without displaying a window with an account selection. I want to be able to reconnect to another metamask account. Natively this is easy to do by calling the 'wallet_requestPermissions' event, but I don't know how to trigger this event using wagmi. 
This is a very important feature for my application beacause it works using the iframe and user can't reconnect using metamask.

**Comment by tmm** [comment]
PR open for this https://github.com/tmm/wagmi/pull/616
